roles/nuage_node/templates/vsp-openshift.j2

```diff
@@ -9,7 +9,7 @@ enterpriseName: {{ enterprise }}
 # Name of the domain in which pods will reside
 domainName: {{ domain }}
 # Name of the VSD user in admin group
-vsdUser: {{ vsduser }}
+vsdUser: {{ vsd_user }}
```

Comment:
I think the current code snippet in my PR is ok as per the requirement. Thanks for the feedback though @sdodson